zpool-iostat-viz

```diff
@@ -225,14 +225,19 @@ def render_stats(window, transform, should_show_differential, pool, filename=Non
         elif in_key == ord('d'):
             should_show_differential = not should_show_differential
         elif in_key == ord('m'):
+            stats = None
+            stats_history = []
```

Comment:
Instead of discarding recent history, I'd prefer it if stats and stats_history were unaffected by views and transforming functions were only used to interpret the raw data.

Reply:
If you do not want to clear the stats_history array, I'd assume that you need two arrays, one for the device centric view, the other for the measurements centric view.
Always applying both transforms and just selecting one of 2 stats and stats_history arrays for display would be a simple change, but my main intend was to provide a fix for the issue that resulted from the addition of the "m" command.
If you provide 2 sets of arrays and put the restriction of "current" to the allowed index range at the start of the loop, then resetting of current to 0 could also be dropped.
I'll update the FreeBSD port to whatever you decide to implement.